### PR TITLE
IIS-Pack Modification (removed Tables and cleaned up info)

### PR DESCRIPTION
**Pull request by pageme27**
Updates to IIS-Pack md file.

### CODE REVIEW

**Comment by mosabua** [comment]
I assume you are going to reintroduce some of the killed content in separate component docs and will therefore leave this PR as "in progress"

**Comment by pageme27** [comment]
Yes, still work in progress. Adding in several component files and adding some additional info to the IIS Pack file.

**Comment by pageme27** [comment]
Adding new component files in this PR relating to the iis-pack: LB, Volume, iis-framework, dotnetframework, chocolatey-package,  nuget-package. Wanted to state this to prevent duplication from others adding. Basic info to be populated. Updating components.md file to link to these new files.

**Comment by mosabua** [comment]
Awesome ... looking forward to it @pageme27

File: user/design/chocolatey-package-component.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: user-doc
+title: Chocolatey Package Component
+---
+
+The _chocolatey-package_ [component](./components.html) is utilized by several packs to install additional software artifacts on
```
Comment (mosabua) [inline user/design/chocolatey-package-component.md]: link to Chocolatey site and mention what it is. Do not assume users know that..

File: user/design/ms-dotnetframework-component.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: user-doc
+title: DotNet Framework Component
+---
+
+The _dotnetframework_ [component](./components.html) is related to multiple Windows/.Net related packs used to configure
+the .Net Framework that is used with the windows-based application will run.
```
Comment (mosabua) [inline user/design/ms-dotnetframework-component.md]: Link to external website about .Net

File: user/design/ms-iis-framework-component.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+layout: user-doc
+title: IIS Framework Component
+---
+
+The _iis-framework_ [component](./components.html) is related to multiple Windows/.Net related packs used to configure how
```
Comment (mosabua) [inline user/design/ms-iis-framework-component.md]: Not true. I think it is the component that manages the configuration for Microsoft IIS ... also link to its website

File: user/design/nuget-package-component.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+layout: user-doc
+title: NuGet Package Component
+---
+
```
Comment (mosabua) [inline user/design/nuget-package-component.md:5]: Link to nuget site and mention what it is .. also link to the package repo

File: user/design/volume-component.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+layout: user-doc
+title: Volume Component
+---
+
+The _volume_ [component](./components.html) is of core importance and part of most platforms since it
```
Comment (mosabua) [inline user/design/volume-component.md]: Do we have more detail? e.g. Mountpoint and drive letter usage ..

**Comment by mosabua** [comment]
Sounds good to me. I will merge later today ..

**Pull request merged**